### ISSUE TITLE
Support for "No further question Skip" to Reply as User task

### ISSUE THREAD

**Issue by Muchaszewski** [LAION-AI/Open-Assistant#1200]
**Description:**

It would be useful to have a feature that allows the Reply as User task to skip over any questions where there is no response to continue the conversation. This can help reduce manual effort in cases where there are many unanswerable questions in a dataset. Right now I have to write in the Skip prompt "Nothing to ask" or similar.

**Expected behavior:**

A button to skip with "No Further Question Skip" in the Reply as User task. When clicked, if there is no answer to a question, the task will skip to the next question.

**Current behavior:**

The Reply as User task currently requires manual add information on why the skip occured over unanswerable questions.

**Steps to reproduce (optional):**

N/A

**Proposal:**

Add a button to Skip with "No Further Question Skip" in the Reply as User task.

**Benefits:**

Reduces manual effort required to skip over unanswerable questions
Improves efficiency in datasets with a high number of unanswerable questions

**Impact:**

Low. This feature would only affect the Reply as User task and would not impact any other functionality.

**Who will be most impacted by this change?**

Users who use the Reply as User task in their workflows will be impacted by this change.

**Comment by AbdBarho** [comment]
So basically end the conversation?

**Comment by Muchaszewski** [comment]
Yes, as Skip does right now. But skip has an optional field that IHMO you should fill in. It could be useful to have this as a predefined option,

Could be also used later on to have options like that
 - Skip with comment
 - Skip no further questions can be asked
 - Skip no knowledge/expertise
 etc.

Example from github on possible GUI ;) 
![image](https://user-images.githubusercontent.com/5092066/216855675-83ad3f6d-3312-4a71-9d73-aefc3157222a.png)

**Comment by fozziethebeat** [comment]
I agree, we need a way to resolve the task.

**Comment by olliestanley** [comment]
I don't think this is necessary. It's very rare that it's not possible to ask any follow-up prompt at all, and skipping is already possible without writing anything in the box
